rosetta-source/src/main/rosetta/event-common-type.rosetta

```diff
@@ -250,6 +252,12 @@ type SettlementOrigin: <"Defines the origin to the transfer as a reference for l
 type TradeIdentifier extends Identifier: <"Defines a trade identifier as a special case of the generic identifier type, that also includes the trade identifier class.">
 	identifierType TradeIdentifierTypeEnum (0..1) <"The enumerated classification of the identifier. Optional as a trade identifier may be party-specific, in which case it may not correspond to any established classification.">	
 
+type TradeValuation: <"Defines as the value of an investment, asset, or security">
+    amount Money (1..1) <"Current value of the outstanding contract">
+    datetime zonedDateTime (1..1) <"Date and time of the last valuation marked to market, provided by the central counterparty (CCP) or calculated using the current or last available market price of the inputs.">
+    method ValuationMethodEnum (1..1) <"Source and method used for the valuation of the transaction by the reporting counterparty.">
```

Comment:
Ditto as above, change reference to 'reporting counterparty' to 'valuation party' instead.